helm/prometheus-meta-operator/templates/rbac.yaml

```diff
@@ -49,6 +49,21 @@ rules:
     verbs:
       - create
       - delete
+  - apiGroups:
```

Comment:
Can you add a todo here to say we need to remove it once the resize is done and not leave some leftover rights?

Reply:
done

Reply:
:1st_place_medal: